data_model/clusters/Messages.xml

```diff
@@ -216,7 +216,7 @@ Davis, CA 95616, USA
         <quality nullable="true"/>
         <mandatoryConform/>
       </field>
-      <field id="4" name="Duration" type="uint16" default="0">
+      <field id="4" name="Duration" type="uint64" default="0">
```

Comment:
This is based on a scrape of the specification (which was done last week).

It is not meant for manual change. Also looking at my copy of the spec:

![image](https://github.com/project-chip/connectedhomeip/assets/1832280/f8213415-e7f4-41bc-8e64-cc7692ad3990)